diff -ruN .gitignore .gitignore
--- .gitignore	1970-01-01 02:00:00
+++ .gitignore	2022-11-27 18:10:18
@@ -0,0 +1,2 @@
+*
+!.gitignore
diff -ruN protocol/tokenization/StableDebtToken.sol protocol/tokenization/StableDebtToken.sol
--- protocol/tokenization/StableDebtToken.sol	2022-09-13 11:40:52
+++ protocol/tokenization/StableDebtToken.sol	2022-11-26 13:07:40
@@ -336,7 +336,8 @@
    * @param avgRate The average rate at which the total supply increases
    * @return The debt balance of the user since the last burn/mint action
    **/
-  function _calcTotalSupply(uint256 avgRate) internal view returns (uint256) {
+  function _calcTotalSupply(uint256 avgRate) virtual internal view returns (uint256) {
+    // Added virtual
     uint256 principalSupply = super.totalSupply();
 
     if (principalSupply == 0) {
